applications/sasquatch/charts/app-metrics/values.yaml

```diff
@@ -0,0 +1,114 @@
+## Default values.yaml for the Metrics Events subchart.
+
+# -- app-metrics configuration in any environment in which the subchart is
+# enabled. This should stay globally specified here, and it shouldn't be
+# overridden. Looks like this:
+#
+# globalAppConfig:
+#   some-app:
+#     influxTags:
+#       - some
+#       - tags
+#   some-other-app:
+#     influxTags:
+#       - some
+#       - tags
+#       - and
+#       - others
+globalAppConfig: {}
+
+# -- A list of applications that will publish metrics events, and the keys that should be ingested into InfluxDB as tags.
+# The names should be the same as the app names in Phalanx.
+apps: []
+
+# -- Keys in an every event sent by any app that should be recorded in InfluxDB
+# as "tags" (vs. "fields").
+#
+# -- These will be concatenated with the `influxTags` from `globalAppConfig`
```

Comment:
It looks like helm-docs ignores the first paragraph and only includes the second. So Perhaps collapse into a single line?